pkg/openapi/openapi_generated.go

```diff
@@ -22,6 +22,7 @@ package openapi
 
 import (
 	spec "github.com/go-openapi/spec"
+	resource "k8s.io/apimachinery/pkg/api/resource"
```

Comment:
doesn't matter, but generated in the non-generated commit.

Reply:
oops...